### PR TITLE
moved pa endpoints

### PR DESCRIPTION
**Pull request by migrenaa**

### CODE REVIEW

File: src/contracts/distributedTown.contracts.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      const contract = distributedTownContract();
+      const isDiToNative = await contract.isDiToNativeCommunity(address);
+      return isDiToNative;
+    } catch (err) {
+      console.log(err);
+    }
+  }
+
+  // TODO: update
+  public static async createCommunity(
```
Comment (migrenaa) [inline src/contracts/distributedTown.contracts.ts]: remove this. it's not used.

File: src/contracts/distributedTown.contracts.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      console.log(result);
+      return result;
+    } catch (err) {
+      console.log(err);
+      return;
+    }
+  };
+
+
+  public static async setPartnersRegistry(
```
Comment (migrenaa) [inline src/contracts/distributedTown.contracts.ts]: remove.

File: src/contracts/distributedTown.contracts.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      const contract = distributedTownContract();
+      const addresses = await contract.getCommunities();
+      console.log(addresses);
+      return addresses;
+    } catch (err) {
+      console.log(err);
+    }
+  }
+
+  public static async getMetadata(tokenID: string) {
```
Comment (migrenaa) [inline src/contracts/distributedTown.contracts.ts]: I believe this is also not used, but please check.

File: src/contracts/distributedTown.contracts.ts (hunk 1 above)
Comment (mladenig) [reply src/contracts/distributedTown.contracts.ts]: [563e29f](https://github.com/SkillWallet/backend/commit/563e29fe1dfbde42f00f353101428983077ef19f)

File: src/contracts/distributedTown.contracts.ts (hunk 2 above)
Comment (mladenig) [reply src/contracts/distributedTown.contracts.ts]: [563e29f](https://github.com/SkillWallet/backend/commit/563e29fe1dfbde42f00f353101428983077ef19f)

File: src/contracts/distributedTown.contracts.ts (hunk 3 above)
Comment (mladenig) [reply src/contracts/distributedTown.contracts.ts]: [563e29f](https://github.com/SkillWallet/backend/commit/563e29fe1dfbde42f00f353101428983077ef19f)

**Pull request merged**